docs/setup/upgrade.asciidoc

```diff
@@ -23,7 +23,7 @@ ifeval::[ "{version}" != "{minor-version}.0" ]
 |Upgrade to {version}
 endif::[]
 
-|7.0–7.13
+|7.0–7.16
```

Comment:
@gchaps We'll need to update this in our backports. For example, in the 7.16 backport, this will become `7.15`, etc.